File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] res = new [2*n];

        for (int i = 0; i < n; i++){
            res[i] = nums[i];
            res[i + n] = nums[i];
        }

        return res;
    }
}

File: Data Structures & Algorithms/max-water-container/submission-1.py
class Solution:
    def maxArea(self, heights: List[int]) -> int:
        res = 0
        left, right = 0, len(heights) - 1

        while left < right:
            curr_area = (right - left) * min(heights[left], heights[right])
            if heights[left] < heights[right]:
                left += 1
            else:
                right -= 1
            res = max(res, curr_area)
        return res